tests/menu_items_registry.test.ts

```diff
@@ -301,6 +303,20 @@ describe("Menu Item actions", () => {
         elements: [3, 4],
       });
     });
+
+    test("cannot delete all non-frozen rows with contextmenu when frozen panes exist", async () => {
+      const ROW_5 = { x: 30, y: 100 };
+      const sheetId = model.getters.getActiveSheetId();
+      const lastRow = model.getters.getNumberRows(sheetId) - 1;
+
+      freezeRows(model, 4, sheetId);
+      selectRow(model, 4, "newAnchor");
+      selectRow(model, lastRow, "updateAnchor");
+
+      simulateContextMenu(".o-row-resizer", ROW_5);
+      await nextTick();
+      expect(fixture.querySelector(".o-menu div[data-name='delete_row']")).toBeNull();
```

Comment:
Change it to use the `getNode(path).isVisible` like the most other tests of this file. The way you did it won't work at the forward port, we optimized this file by not mounting the spreadsheet, disallowing the use of contextClicks to open the menu

Reply:
Changes made